payments/credit-card/index.md

```diff
@@ -0,0 +1,514 @@
+* PayEx will handle 3D-secure authentication when this is required.  
+* PayEx will redirect the payer's browser to - or display directly in  the iFrame - one of two specified URLs, depending on whether the payment session is followed through completely or cancelled beforehand. Please note that both a successful and rejected payment reach completion, in contrast to a cancelled payment.  
+* When you detect that the payer reach your completeUrl , you need to do a `GET` request, containing the paymentID generated in the first step, to receive the state of the transaction.  
+
+### Payment Url
+
+For our hosted view, the URL property called paymentUrl will be used if the consumer is redirected out of the hosted view frame through our [Credit Card API](https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/technical-reference/core-payment-resources/card-payments/). The consumer is redirected out of frame when at the 3d secure verification for credit card payments. The URL should represent the page of where the payment hosted view was hosted originally, such as the checkout page, shopping cart page, or similar. Basically, paymentUrl should be set to the same URL as that of the page where the JavaScript for the hosted payment view was added to in order to initiate the payment. Please note that the paymentUrl must be able to invoke the same JavaScript URL from the same Payment as the one that initiated the payment originally, so it should include some sort of state identifier in the URL. The state identifier is the ID of the order, shopping cart or similar that has the URL of the Payment stored.
+
+
+With paymentUrl in place, the retry process becomes much more convenient for both the integration and the payer.
+
+## Screenshots
+
+You will redirect the consumer/end-user to PayEx hosted pages to collect the credit card information.
+
+![Redirect image](https://developer.payex.com/xwiki/wiki/developer/download/Main/ecommerce/payex-payment-instruments/card-payments/card-payment-pages-in-mobile-apps/WebHome/1551693930590-433.png?width=400&height=711)
+
+## API-requests
+
+The API requests are displayed in the[ purchase flow](https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/payex-payment-instruments/card-payments/card-payment-pages-in-mobile-apps/#HPurchaseflow). The options you can choose from when creating a payment with key operation set to Value Purchase are listed below. The general REST based API model is described in the [technical reference](https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/technical-reference/).
```

Comment:
We shouldn't have links to `developer.payex.com`. Use `#` if what's being linked to doesn't yet exist on `developer.swedbankpay.com`.